web/styles/inputs.css

```diff
@@ -0,0 +1,86 @@
+.input-element-wrapper {
+    display: grid;
+    grid-template: "icon input button" auto / auto minmax(0, 1fr) auto;
+    grid-template-rows: 1fr;
+    align-items: center;
+}
+
+.input-icon {
+    pointer-events: none;
+    grid-area: icon;
+    margin-left: 0.5em;
```

Comment:
Here and throughout, as always, we should be tracking the px-to-em conversions in short CSS comments.

Reply:
Added px-to-em conversions throughout the changes.